Rx/v2/src/rxcpp/rx-operators.hpp

```diff
@@ -97,8 +97,6 @@ class operator_factory
 
 #include "operators/rx-connect_forever.hpp"
 #include "operators/rx-lift.hpp"
-#include "operators/rx-multicast.hpp"
-#include "operators/rx-publish.hpp"
```

Comment:
I would like to think about these dependencies on publish/multicast longer.

Until then, please try the build with the original include structure (add these two includes back and remove all the includes of multicast and publish that were added).

Thank you!